secureli/modules/core/core_services/scanner.py

```diff
@@ -2,11 +2,15 @@
 from typing import Optional
 from pathlib import Path
 
-import pydantic
 import re
 
 from secureli.modules.shared.abstractions.pre_commit import PreCommitAbstraction
-from secureli.modules.shared.models.scan import ScanFailure, ScanMode, ScanResult
+from secureli.modules.shared.models.scan import (
+    ScanFailure,
+    ScanMode,
+    ScanOutput,
+    ScanResult,
+)
```

Comment:
From previous refactor work @JordoHeffernan suggested that it is more pythonic to import 
```python 
from secureli.modules.shared.models import scan
```

and then you can use `scan.ScanFailure`, `scan.ScanMode`, `scan.ScanOutput`, and `scan.ScanResult`.

Reply:
TIL. Can do!